lib/charts/chart.rb

```diff
@@ -1,8 +1,8 @@
 class Charts::Chart
   attr_reader :data,
-              :options,
-              :prepared_data,
-              :renderer
+    :options,
+    :prepared_data,
+    :renderer
 
```

Comment:
Look at the first method name, it is way off to the right from where these are.  The colons should be aligned, for ease of reading.